models/balancer/ethereum/balancer_ethereum_balances.sql

```diff
@@ -0,0 +1,71 @@
+{{
+    config(
+        alias='balancer_balances',
```

Comment:
since the schema is ```balancer_ethereum``` based on your dbt_profile edits, the table alias can just be ```balances``` for simplicity

Reply:
Fixed.

Changed the table alias to `balances`.